src/main/webapp/app/components/tabs/CurationDataValidationTab.tsx

```diff
@@ -0,0 +1,330 @@
+    if (x.type !== validationType) return false;
+    const noFilterFlags = !filter.success && !filter.error;
+    const successCheck = noFilterFlags || (filter.success && x.status === 'IS_COMPLETE');
+    const errorCheck = noFilterFlags || (filter.error && x.status === 'IS_ERROR');
+    return successCheck || errorCheck || x.status === 'IS_PENDING';
+  });
+
+  validationResults.sort((a, b) => {
+    if ((a.status === 'IS_ERROR' && b.status !== 'IS_ERROR') || (a.status === 'IS_COMPLETE' && b.status === 'IS_PENDING')) {
+      return -1;
+    } else if ((a.status !== 'IS_ERROR' && b.status === 'IS_ERROR') || (a.status === 'IS_PENDING' && b.status === 'IS_COMPLETE')) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
+
+  return (
+    <div className="d-flex flex-column" style={{ gap: '0.25rem' }}>
+      {validationResults.map(validationResult => {
+        const disableCollapsible = validationResult.data.length === 0;
+        const { status } = validationResult;
+        const color = status === 'IS_COMPLETE' ? SUCCESS : status === 'IS_ERROR' ? DANGER : GREY;
+        const showLoadingSpinner = status === 'IS_PENDING';
+        const isLarge = validationResult.data.length > 10;
+        return (
+          <Collapsible
+            title={validationResult.key}
+            colorOptions={{
+              borderLeftColor: color,
+              forceLeftColor: true,
+            }}
+            showLoadingSpinner={showLoadingSpinner}
+            displayOptions={{
+              disableCollapsible,
+            }}
+            key={validationResult.key}
+          >
+            <div className="py-3">
+              <OncoKBTable
+                data={validationResult.data}
+                columns={validationColumns}
+                showPagination={isLarge}
+                disableSearch={!isLarge}
+                defaultSorted={[
+                  {
+                    id: 'target',
+                    desc: false,
+                  },
+                ]}
+              />
+            </div>
+          </Collapsible>
+        );
+      })}
+    </div>
+  );
+}
+
+function CurationDataValidationTab() {
+  const [validationResultFilter, setValidationResultFilter] = React.useState<ValidationResultFilter>({
+    success: false,
+    error: false,
+  });
+
+  const [isValidating, setIsValidating] = React.useState(false);
+  const [validationCounter, setValidationCounter] = React.useState(0);
+
+  useEffect(() => {
+    setIsValidating(validationCounter > 0);
+  }, [validationCounter]);
+
+  const onClose = useCallback(() => {
+    setIsValidating(false);
+    notifySuccess('Validation completed!');
+  }, []);
+  const onOpen = useCallback(() => {
+    return;
+  }, []);
+  const onError = useCallback(() => {
+    notifyError(new Error('Error while attempting to validate. Please try again.'));
+  }, []);
+  const url = relativeWebsocketUrlToAbsolute('/api/websocket/curation/validation');
```

Comment:
I would prefer creating a websocket API in curation platform and let the server side routes certain URL pattern to the core service so we could also add other websockets as well.